bucket/featherwallet.json

```
@@ -0,0 +1,39 @@
+{
+    "version": "1.0.1",
+    "description": "A Monero SPV Wallet",
+    "homepage": "https://featherwallet.org/",
+    "license": {
+        "url": "https://git.featherwallet.org/feather/feather/raw/branch/master/LICENSE"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://featherwallet.org/files/releases/win/feather-1.0.1-win.zip",
+            "hash": "aa5f0a38d6258af2deb01c5c5c35acbc1e1311882357e7996c3022bff48932fe"
+        }
+    },
+    "pre_install": [
+        "Move-Item \"$dir\\feather*.exe\" \"$dir\\feather.exe\"",
```

```suggestion
        "Rename-Item \"$dir\\feather-$version.exe\" \"$dir\\feather.exe\"",
```
In response to @pratikpc's comment

---

this is actually how i approached it originally but didn't know i was allowed to use $version in there. thanks!